src/AdminConsole/Pages/Shared/_AddPasskeys.cshtml

```diff
@@ -18,9 +24,9 @@
         </fieldset>
     </form>
 
-    <div class="status mt-3" v-cloak>
-        <SuccessAlertBox v-if="feedback.status =='completed'" message="{{feedback.message}}"></SuccessAlertBox>
-        <DangerAlertBox v-if="feedback.status == 'error'" message="{{feedback.message}}"></DangerAlertBox>
+    <div asp-if="hasCredentialAddFeedback" class="status mt-3" v-cloak>
+        <success-alert-box asp-if="@(credentialAddStatus == "completed")" message="@credentialAddMessage"></success-alert-box>
+        <danger-alert-box asp-if="@(credentialAddStatus == "error")" message="@credentialAddMessage"></danger-alert-box>
```

Comment:
Hmm, is there another way to set the message that does not involve reading data provided in the URL? It's a foot gun when it comes to introducing XSS vulnerabilities and would be easier to just avoid it rather than doing the right encoding (and maintaining that encoding).

Reply:
@abergs It would involve redesigning how it works completely. Because you also need to refresh the list of credentials. I did now add HtmlSanitizer.